src/Inventory.js

```diff
@@ -6,17 +6,15 @@ class Inventory extends Map {
    */
   addItem(item) {
     this.set(item.uuid, item);
-    item.isHeld = true;
-    item.container = this;
+    item.belongsTo = this;
```

Comment:
Hrm. That begs the question, how do I get that from inside this Inventory object? Should the Inventory have a reference to the thing it belongs to as well?